pypif/pif.py

```diff
@@ -26,51 +26,53 @@ def dumps(pif, **kwargs):
     return json.dumps(pif, cls=PifEncoder, **kwargs)
 
 
-def load(fp, **kwargs):
+def load(fp, builder=None, **kwargs):
     """
     Convert content in a JSON-encoded text file to a Physical Information Object or a list of such objects.
 
     :param fp: File-like object supporting .read() method to deserialize from.
     :param kwargs: Any options available to json.load().
     :return: Single object derived from :class:`.Pio` or a list of such object.
     """
-    return loado(json.load(fp, **kwargs))
+    return loado(json.load(fp, **kwargs), builder=builder)
 
 
-def loads(s, **kwargs):
+def loads(s, builder=None, **kwargs):
```

Comment:
Can you add to the comments for each method and give the type of builder as well as its purpose.